monai/apps/auto3dseg/data_analyzer.py

```diff
@@ -169,6 +173,58 @@ def _check_data_uniformity(keys: list[str], result: dict) -> bool:
         return True
 
     def get_all_case_stats(self, key="training", transform_list=None):
+        """ Wrapper for the internal _get_all_case_stats to perform multi-gpu processing
+        """
+        if self.device.type == 'cpu':
+            nprocs = 1
```

Comment:
Will this negate the multi-processing ability of using CPU?

Reply:
no. the single cpu process can still use multiple workers

Reply:
I am re-opening this thread as I have another question. Is there any performance loss to initiate the manager for CPU/single GPU, which is not necessary?

Reply:
Yes. The multiprocessing initialization will use 10s. I changed the code so CPU/single GPU will not have multiprocessing initializaed.